pkg/controller/broker/v2alpha2/activemqartemis/upgrade.go

```diff
@@ -43,13 +41,11 @@ func checkVersion(productVersion string) bool {
 	return false
 }
 
-// getMinorImageVersion ...
 func getMinorImageVersion(productVersion string) string {
 	major, minor, _ := MajorMinorMicro(productVersion)
 	return strings.Join([]string{major, minor}, "")
 }
 
-// MajorMinorMicro ...
```

Comment:
I didn't realize there were so many of these done this way... likely for lint check, which seems to be disabled in makefile right now. I could have mislead you in removing it, but I do feel like having an empty placeholder just to pass a lint check would defeat the purpose. That's for the project team to decide, though :)